l1-contracts/contracts/common/libraries/openzeppelin/IncrementalMerkle.sol

```diff
@@ -26,9 +26,6 @@ library DynamicIncrementalMerkle {
      *
      * The `sides` and `zero` arrays are set to have a length equal to the depth of the tree during setup.
      *
-     * The hashing function used during initialization to compute the `zeros` values (value of a node at a given depth
```

Comment:
This modifies a library in the folder for openzeppeling ones. We should then probably add a comment that it was modified compared to the original and list changes. Otherwise, we give false perception that this code is exactly as original and should not be audited